2017-06-05 13:24:16 [scrapy.utils.log] INFO: Scrapy 1.4.0 started (bot: greenbook)
2017-06-05 13:24:16 [scrapy.utils.log] INFO: Overridden settings: {'BOT_NAME': 'greenbook', 'FEED_FORMAT': 'csv', 'FEED_URI': 'spidergod9.csv', 'LOG_FILE': 'scrapy8.log', 'NEWSPIDER_MODULE': 'greenbook.spiders', 'ROBOTSTXT_OBEY': True, 'SPIDER_MODULES': ['greenbook.spiders']}
2017-06-05 13:24:16 [scrapy.middleware] INFO: Enabled extensions:
['scrapy.extensions.corestats.CoreStats',
 'scrapy.extensions.telnet.TelnetConsole',
 'scrapy.extensions.feedexport.FeedExporter',
 'scrapy.extensions.logstats.LogStats']
2017-06-05 13:24:16 [scrapy.middleware] INFO: Enabled downloader middlewares:
['scrapy.downloadermiddlewares.robotstxt.RobotsTxtMiddleware',
 'scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware',
 'scrapy.downloadermiddlewares.downloadtimeout.DownloadTimeoutMiddleware',
 'scrapy.downloadermiddlewares.defaultheaders.DefaultHeadersMiddleware',
 'scrapy.downloadermiddlewares.useragent.UserAgentMiddleware',
 'scrapy.downloadermiddlewares.retry.RetryMiddleware',
 'scrapy.downloadermiddlewares.redirect.MetaRefreshMiddleware',
 'scrapy.downloadermiddlewares.httpcompression.HttpCompressionMiddleware',
 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware',
 'scrapy.downloadermiddlewares.cookies.CookiesMiddleware',
 'scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware',
 'scrapy.downloadermiddlewares.stats.DownloaderStats']
2017-06-05 13:24:16 [scrapy.middleware] INFO: Enabled spider middlewares:
['scrapy.spidermiddlewares.httperror.HttpErrorMiddleware',
 'scrapy.spidermiddlewares.offsite.OffsiteMiddleware',
 'scrapy.spidermiddlewares.referer.RefererMiddleware',
 'scrapy.spidermiddlewares.urllength.UrlLengthMiddleware',
 'scrapy.spidermiddlewares.depth.DepthMiddleware']
2017-06-05 13:24:16 [scrapy.middleware] INFO: Enabled item pipelines:
[]
2017-06-05 13:24:16 [scrapy.core.engine] INFO: Spider opened
2017-06-05 13:24:16 [scrapy.extensions.logstats] INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2017-06-05 13:24:16 [scrapy.extensions.telnet] DEBUG: Telnet console listening on 127.0.0.1:6023
2017-06-05 13:24:37 [scrapy.downloadermiddlewares.retry] DEBUG: Retrying <GET http://www.thegreenbook.com/robots.txt> (failed 1 times): TCP connection timed out: 10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond..
2017-06-05 13:24:58 [scrapy.downloadermiddlewares.retry] DEBUG: Retrying <GET http://www.thegreenbook.com/robots.txt> (failed 2 times): TCP connection timed out: 10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond..
2017-06-05 13:25:15 [scrapy.crawler] INFO: Received SIGINT, shutting down gracefully. Send again to force 
2017-06-05 13:25:15 [scrapy.core.engine] INFO: Closing spider (shutdown)
2017-06-05 13:25:16 [scrapy.extensions.logstats] INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2017-06-05 13:25:17 [scrapy.crawler] INFO: Received SIGINT twice, forcing unclean shutdown
2017-06-05 13:25:17 [scrapy.downloadermiddlewares.retry] DEBUG: Gave up retrying <GET http://www.thegreenbook.com/robots.txt> (failed 3 times): An error occurred while connecting: [Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.ConnectionLost'>: Connection to the other side was lost in a non-clean fashion: Connection lost.
].
2017-06-05 13:25:17 [scrapy.downloadermiddlewares.robotstxt] ERROR: Error downloading <GET http://www.thegreenbook.com/robots.txt>: An error occurred while connecting: [Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.ConnectionLost'>: Connection to the other side was lost in a non-clean fashion: Connection lost.
].
Traceback (most recent call last):
  File "c:\users\darren\anaconda3\lib\site-packages\twisted\internet\defer.py", line 1299, in _inlineCallbacks
    result = result.throwExceptionIntoGenerator(g)
  File "c:\users\darren\anaconda3\lib\site-packages\twisted\python\failure.py", line 393, in throwExceptionIntoGenerator
    return g.throw(self.type, self.value, self.tb)
  File "c:\users\darren\anaconda3\lib\site-packages\scrapy\core\downloader\middleware.py", line 43, in process_request
    defer.returnValue((yield download_func(request=request,spider=spider)))
twisted.internet.error.ConnectError: An error occurred while connecting: [Failure instance: Traceback (failure with no frames): <class 'twisted.internet.error.ConnectionLost'>: Connection to the other side was lost in a non-clean fashion: Connection lost.
].
